src/inference/dev_api/threading/ie_istreams_executor.hpp

```diff
@@ -74,6 +74,7 @@ class INFERENCE_ENGINE_API_CLASS(IStreamsExecutor) : public ITaskExecutor, publi
             const bool enable_hyper_thread = true);  // no network specifics considered (only CPU's caps);
         static int GetHybridNumStreams(std::map<std::string, std::string>& config, const int stream_mode);
         static void UpdateHybridCustomThreads(Config& config);
+        static Config ReserveCpuCores(Config& config);
```

Comment:
remove, we have not agreed to have this method on arch forum

Reply:
Ok, done.